bedrock/teach/templates/teach/smarton/index.html

```diff
@@ -0,0 +1,86 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on the Web') }}{% endblock %}
+{% block page_desc %}{{ _('The SmartOn Series is like your Internet owner’s manual: the place to learn the most useful intel and tips from Mozilla policy peeps and programmers.') }}{% endblock %}
+{% block body_id %}smarton-landing{% endblock %}
+{% block body_class %}smarton landing{% endblock %}
+
+{% block smarton_content %}
+<main role="main" class="section">
+  <div class="content">
+
+    <ul class="smarton-topics">
+      <li class="topic topic-tracking" id="topic-tracking" data-path-hide="M0,0 v260 c0,0,60,6,130-5 c80-10,130-5,130-5 V0 H0 z" data-path-show="M0,0 v70 c0,0,45,14,130,14 s130-14,130-14 V0 H0 z">
+        <h2 class="topic-title">{{ _('Tracking') }}</h2>
+        <p class="topic-intro">{{ _('What you do online is your business, and you can keep it that way.') }}</p>
+        <a href="{{ url('teach.smarton.tracking') }}" class="inner">
+          <p class="cta">
+          {% if l10n_has_tag('smarton-surveillance') %}
+            {{ _('Outfox the trackers') }}
```

Comment:
@flodolo : Some late string changes on the landing page behind the tag 'smarton-surveillance'